testing/kuttl/e2e/exporter-password-change/02-assert.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+# Ensure that the password has been updated in the exporter and it can still access
+# Postgres. 
+# - Check that the exporter pid has changed
+# - Check that the DATA_SOURCE_PASS_FILE contains the expected password (`password`)
+# - Check that all containers in the instance pod are ready after restart (TODO: review comment after watcher)
```

Comment:
Can we do that TODO?

Reply:
With the watcher changes, the containers should ever be "not ready". I'm thinking it might be possible to remove that section of the test. I don't see what it tells us